### PR TITLE
added event listener and new js file

### PR DESCRIPTION
**Pull request by sonyco-4u**
#interactive list

add some functionality to my to-do list to make it interactive.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @sonyco-4u, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have enough files changed in files changed for me to make a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Deepakdanger — APPROVED** [review]
Hi @sonyco-4u , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
+      </header>
+      <form action="#" class="d-flex s-between" id="add-task" method="post">
+          <input type="text" name="activity" placeholder="Add your List">
+          <button type="submit">
+              <span class="material-icons">keyboard_return</span>
+          </button>
+      </form>
+      <ul id="list-items">
+
+      </ul>
```
Comment (sonyco-4u) [inline dist/index.html:25]: Applying the html best-practices
- On the index.html file on dist directory I notice there is some two double empty spaces left.

File: src/list.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    // this code is for the delete activity button
+    const deleteBtns = document.querySelectorAll('.delete-activity');
+    if (deleteBtns) {
+      deleteBtns.forEach((activity) => {
+        activity.addEventListener('click', () => {
+          this.deleteActivity(activity.getAttribute('data'));
+        });
+      });
+    }
+    // this code edits the activity handler
```
Comment (sonyco-4u) [inline src/list.js:123]: Use space-less equal signs, in other to follow JavaScript best practice

use <div class="striped"> not <div class = "striped">

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  width: calc(100% - 2rem);
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+form input {
+  width: calc(100% - 25px);
+  border: 0 solid transparent;
+  outline: none;
```
Comment (sonyco-4u) [inline src/style.css:77]: In following CSS best practice always 
Be careful with blank lines and indentation.
Do not add blank lines, spaces, or indentations without a reason.
For readability, add blank lines to separate large or logical code blocks.

File: test/list.test.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  });
+
+  describe('Updating a task', () => {
+    test('Update task status', () => {
+      const todoList = new List();
+      todoList.addActivity('Test');
+      todoList.updateActivityStatus(1);
+      expect(todoList.list[0].completed).toBe(true);
+      todoList.clearAll();
+    });
```
Comment (sonyco-4u) [inline test/list.test.js:59]: Remeber to follow the following rules **(DRY, KISS, and YAGNI)** in all your code including CSS, HTML and JavaScript.

**Review by sonyco-4u — COMMENTED** [review]
### **Great job so far!**
please check all my comments below and make all the necessary corrections .

 submit for a re-review. when you are done. 👍

**Pull request merged**